RadarSDK/Radar.m

```diff
@@ -252,8 +251,11 @@ + (void)getContextWithCompletionHandler:(RadarContextCompletionHandler)completio
     [[RadarLocationManager sharedInstance] getLocationWithCompletionHandler:^(RadarStatus status, CLLocation *_Nullable location, BOOL stopped) {
         if (status != RadarStatusSuccess) {
             if (completionHandler) {
-                return completionHandler(status, nil, nil);
+                [RadarUtils runOnMainThreadAsyncIfNecessary:^{
+                    completionHandler(status, nil, nil);
+                }];
             }
+            return;
```

Comment:
out of curiosity, why do the `getLocation` completion handlers have a `return`, but the main ones with the API result do not?

Reply:
just an early return, to prevent calling `RadarAPIClient` when the `getLocation` errors out